docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md

```diff
@@ -92,8 +92,14 @@ To connect the board:
     !!! tip
         To show available subcommands, use the **Tab** key.
 
-0. Optionally, change the default credentials by following the instructions in the program.
+0. Check your default credentials by following the instructions in the program.
     ![Anjay configuration](images/anjay_config.png "Anjay configuration")
+
+    !!! note
+        If your default credentials are different than device credentials from Coiote IoT Device Management, please change them with `anjay config set possible_option` command.
```

Comment:
```suggestion
        If your default credentials are different than device credentials from Coiote IoT Device Management, please change them with `anjay config set <possible_option> <value>` command.
```  
It's a good practice to show the reader that some values needs to be changed depending on the situation. In this case, a user needs to change `possible_option` and provide proper `value`.